app/src/main/java/com/poupa/vinylmusicplayer/dialogs/AddToPlaylistDialog.java

```diff
@@ -58,9 +60,43 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                         CreatePlaylistDialog.create(songs).show(getActivity().getSupportFragmentManager(), "ADD_TO_PLAYLIST");
                     } else {
                         materialDialog.dismiss();
-                        PlaylistsUtil.addToPlaylist(getActivity(), songs, playlists.get(i - 1).id, true);
+                        Context ctx = getActivity();
+                        if (hasDuplicates(playlists.get(i - 1).id, songs, ctx)) {
```

Comment:
can `i` be equal to zero?

Reply:
I'd guess it can't be based on the previous behavior also using i-1 there, but I can double check at some point.

Reply:
We've handled the case where `i == 0` already on lines 58-60, so at this point it cannot be zero.